understanding/21/label-in-name.html

```diff
@@ -61,12 +65,16 @@ <h3>Accessible Name and Description Computation specification</h3>
  <p>Other text displayed on the screen that is correctly coded to meet 1.3.1: Info and Relationships is <strong>not</strong> normally factored into the calculation for the accessible name of a UI component without author intervention (via ARIA labeling techniques). The most common of these are:</p>
 	<ul>
 		<li>headings and instructions</li>
-<li>group labels for sets of components (i.e., used with <code>legend</code>/<code>fieldset</code> or with role of <code>group</code> or <code>radiogroup</code>)</li> 
+<li>group labels for sets of components (i.e., used with <code>legend</code>/<code>fieldset</code> or with role of <code>group</code> or <code>radiogroup</code>)</li>
 </ul>
 <p>Such textual information may constitute part of the component's <em>description</em>. So from both a programmatic viewpoint, and from the conservative tactic of only considering a label to be "adjacent text," neither headings, instructions, nor group 'labels' should normally be considered <strong>labels</strong> for the purpose of this Success Criterion.</p>
  <p>It is important to note that the specification allows authors to override the name calculated through native semantics. Both <code>aria-label</code> and <code>aria-labelledby</code> take precedence in the name calculation, overriding the visible text as the accessible name even when the visible text label is programmatically associated with the control. For this reason, when a visible label already exists, <code>aria-label</code> should be avoided or used carefully, and <code>aria-labelledby</code> should be used as a supplement with care.</p>
 <p>Finally, <code>aria-describedby</code> is not included in the Accessible Name computation (instead it is part of the Accessible Description computation). By convention, text associated with a control through <code>aria-describedby</code> is announced immediately after the accessible name by screen readers. Therefore, the context of headings, instructions, and group labels can be provided through the accessible description to assist users of screen readers without affecting the experience of those who navigate using speech recognition software.</p>
-			</section>
+</section>
+<h4>Parenthetical text</h4>
```

Comment:
Even if parenthetical content is the most common case, would it be better to indicate that there may be different ways to visually separate additional content that may then not need not be considered as part of the accname? For example, additional content may be visually separated by different formatting (label bold, additional text normal or italic); separation by a vertical bar; additional text placed  on a new line etc. etc. Should we really constrain this to parentheses only?

Reply:
@detlevhfischer Parentheses are the common use case. I'm not aware of text styling differences being treated differently by speech input users, and I have not seen the use of a separator. Do you have examples that you have in mind?
Ultimately, we're treading on soft ground here.  I'm inclined to keep the consideration contained and essential.

We already have guidance about text in **close** proximity being a primary driver, so I don't think material on a new line needs to be listed here.